09.Lesson_HighLevelProperties/ThinkingPropertiesExercise/README.md

```diff
@@ -26,7 +26,13 @@ Don't worry about writing the properties in CVL. For now, we only practice the i
 
 </br>
 
-- [ ] If possible, reach out to a fellow participant of the on-boarding and ask them to discuss your properties. Exchange thoughts and ideas.
+- [ ] Start with the exampleSpec.spec file and write more properties. Focus on multi-contract
+
+- [ ] Learn about [ghosts](https://docs.certora.com/en/latest/docs/cvl/ghosts.html?highlight=ghost)
```

Comment:
Let's add https://docs.certora.com/en/latest/docs/cvl/ghosts.html instead of a current link. The current link highlights the word "ghost," and you need to edit the link manually to get rid of it. Inconvenient.